src/lib/apputils/net-server.c

```diff
@@ -1329,7 +1336,9 @@ process_tcp_connection_read(verto_ctx *ctx, verto_ev *ev)
     } else {
         /* msglen known. */
         socklen_t local_saddrlen = sizeof(struct sockaddr_storage);
-        struct sockaddr *local_saddrp = NULL;
+        krb5_address local_addr_buf;
+        krb5_fulladdr local_addr;
```

Comment:
I've added it to the connection struct now. Once this goes in we should probably first remove the pointer for the address in krb5_fulladdr. After that cleanup will be easier and also copying addresses around.